apps/sms/views/conversation/index.html

```diff
@@ -41,9 +41,10 @@
   <link rel="localization" href="/locales/sms.{locale}.properties">
   <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
   <link rel="localization" href="/shared/locales/intl/unitformat.{locale}.properties">
+  <script defer src="/mocks/navigator_moz_icc_manager.js"></script>
   <!-- Firefox Desktop shims -->
   <script defer src="/desktop-mock/navigator_moz_has_pending_message.js"></script>
-  <script defer src="/desktop-mock/navigator_moz_icc_manager.js"></script>
+  <script defer src="/desktop-mock/navigator_moz_set_message_handler.js"></script>
```

Comment:
Followed your suggestion and moved all system messages related mocks to navigator_moz_system_messages.js :)